sdk/openai/openai/src/OpenAIClient.ts

```diff
@@ -1,6 +1,14 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
+/**
```

Comment:
Why is generated code landing under both the `sources/generated` folder and the `src` folder?

Reply:
The `src` directory is the one that is shipped. The code under `sources` is used to synthesize the code under `src` by combining `sources/generated` and `sources/custom` using the `rushx customize` command.